src/contracts/core/RewardsCoordinator.sol

```diff
@@ -395,16 +407,11 @@ contract RewardsCoordinator is
      * @param activatedAt The timestamp when the split is activated.
      */
     function _setOperatorSplit(OperatorSplit storage operatorSplit, uint16 split, uint32 activatedAt) internal {
```

Comment:
Should we do validation in the setter?

Reply:
Yup I was thinking exactly the same

Reply:
Refactor coming in

Reply:
Done in [b2f8bdd](https://github.com/Layr-Labs/eigenlayer-contracts/pull/947/commits/b2f8bdd1c452c037f9db385fe529fb08da2d8ad8)